src/Directory.Build.targets

```diff
@@ -44,6 +44,13 @@
 
   </ItemGroup>
 
+  <!-- Following found during vulerabilities Code Scan  -->
+  <ItemGroup>
+    <PackageReference Update="System.Net.Http" Version="4.3.4" />
+    <PackageReference Update="System.Private.Uri" Version="4.3.2" />
+    <PackageReference Update="System.Net.RegularExpressions" Version="4.3.1" />
```

Comment:
This should be `System.Text.RegularExpressions`, based on your report in #112

Reply:
I fixed this in the latest push